src/main/java/seedu/address/logic/trie/Trie.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.logic.trie;
+
+import java.util.Set;
+
+/**
+ * Interface for the Command Trie
```

Comment:
header command should be a bit more specific, what do you plan to achieve with this `Trie`?